src/slack.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const AWS = require('aws-sdk');
+const Slack = require('slack-node');
+
+const logger = require('./logger');
+
+const instance = new Slack(process.env.ACCESS_TOKEN);
+
+const uploadToS3 = async (key, metadata, tmpPath) => {
+  const s3 = new AWS.S3();
+
+  try {
+    return s3
```

Comment:
this needs to either await/try/catch here or where it is used, or add a catch here but currently this try catch won't do anything with this promise failing